x-pack/legacy/plugins/apm/public/components/app/Settings/AgentConfigurations/AddEditFlyout/SettingsSection.tsx

```diff
@@ -0,0 +1,174 @@
+  captureBody: string;
+  setCaptureBody: (value: string) => void;
+
+  // transactionMaxSpans
+  transactionMaxSpans: string;
+  setTransactionMaxSpans: (value: string) => void;
+  isTransactionMaxSpansValid?: boolean;
+}
+
+export function SettingsSection({
+  isRumService,
+
+  // sampleRate
+  sampleRate,
+  setSampleRate,
+  isSampleRateValid,
+
+  // captureBody
+  captureBody,
+  setCaptureBody,
+
+  // transactionMaxSpans
+  transactionMaxSpans,
+  setTransactionMaxSpans,
+  isTransactionMaxSpansValid
+}: Props) {
+  return (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          {i18n.translate(
+            'xpack.apm.settings.agentConf.flyOut.settingsSection.title',
+            { defaultMessage: 'Options' }
+          )}
+        </h3>
+      </EuiTitle>
+
+      <EuiSpacer size="m" />
+
+      <EuiFormRow
+        label={i18n.translate(
+          'xpack.apm.settings.agentConf.flyOut.settingsSection.sampleRateConfigurationInputLabel',
+          { defaultMessage: 'Transaction sample rate' }
+        )}
+        helpText={i18n.translate(
+          'xpack.apm.settings.agentConf.flyOut.settingsSection.sampleRateConfigurationInputHelpText',
+          {
+            defaultMessage:
+              'Choose a rate between 0.000 and 1.0. Default is 1.0 (100% of traces).'
+          }
+        )}
+        error={i18n.translate(
+          'xpack.apm.settings.agentConf.flyOut.settingsSection.sampleRateConfigurationInputErrorText',
+          { defaultMessage: 'Sample rate must be between 0.000 and 1' }
+        )}
+        isInvalid={!isEmpty(sampleRate) && !isSampleRateValid}
+      >
+        <EuiFieldText
+          placeholder={i18n.translate(
+            'xpack.apm.settings.agentConf.flyOut.settingsSection.sampleRateConfigurationInputPlaceholderText',
+            { defaultMessage: 'Set sample rate' }
+          )}
+          value={sampleRate}
+          onChange={e => {
+            e.preventDefault();
+            setSampleRate(e.target.value);
+          }}
+        />
+      </EuiFormRow>
+
+      <EuiSpacer size="m" />
+
+      {!isRumService && (
+        <EuiFormRow
+          label={i18n.translate(
+            'xpack.apm.settings.agentConf.flyOut.settingsSection.captureBodyInputLabel',
+            { defaultMessage: 'Capture body' }
+          )}
+          helpText={i18n.translate(
+            'xpack.apm.settings.agentConf.flyOut.settingsSection.captureBodyInputHelpText',
+            {
+              defaultMessage:
+                'For transactions that are HTTP requests, the agent can optionally capture the request body (e.g. POST variables). Default is "off".'
+            }
+          )}
+        >
+          <SelectWithPlaceholder
+            placeholder={i18n.translate(
+              'xpack.apm.settings.agentConf.flyOut.settingsSection.captureBodyInputPlaceholderText',
+              { defaultMessage: 'Select option' }
+            )}
+            options={[
+              { value: 'off', text: 'off' },
```

Comment:
Translations?

Reply:
I talked to @formgeist and we decided not to translate these. I think I can simplify this to just:
```json
[{value: 'off'},...]
```